### ISSUE TITLE
Calendar: Information about the change of week is not narrated by screen reader whenever user press Enter on previous/next button.

### ISSUE THREAD

**Issue by dzearing** [OfficeDev/office-ui-fabric-react#11142]
Tracking all of the issues brought up with the 03/15 accessibility pass on Calendar fabric control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8cfe67c5-ce76-4083-b60e-4190dbd38206%7D&action=default&uid=%7B8CFE67C5-CE76-4083-B60E-4190DBD38206%7D&ListItemId=1872&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

**Comment by lorejoh12** [comment]
This is a bug with the example page, not the control, although we can mitigate this issue with a change to the control.

Right now, the control only announces the navigated state. Month and year. So when the month and year change (either by clicking the arrows within the control itself, or by clicking the example's buttons below the control enough times to change the month), the data gets announced. However, there's no aria-live for the selected date itself.

The simplest fix is to just put aria-live on the selected date in the label for the control. In terms of actually using this component, that means we're expecting the consumer to keep track of when a date gets selected and tell the user about it with their own aria states.

A better fix might be to add an "invisible" aria-live label at the root of the control with the selected date information in it. That way, the immediately obvious visual information of the blue highlight is given to screen reader users as soon as they focus into the control, and when the selection changes, it gets announced.

**Comment by lorejoh12** [comment]
For some reason the PR merging didn't auto close the issues... closing, this should be fixed now.
